diff --git a/src/Resources/MailResource/Pages/ListMails.php b/src/Resources/MailResource/Pages/ListMails.php
--- a/src/Resources/MailResource/Pages/ListMails.php
+++ b/src/Resources/MailResource/Pages/ListMails.php
@@ -29,9 +29,9 @@ class ListMails extends ListRecords
         return $result;
     }
 
-    public function unmountAction(bool $canCancelParentActions = true): void
+    public function unmountAction(bool | string | null $cancelParentActions = null): void
     {
-        parent::unmountAction($canCancelParentActions);
+        parent::unmountAction($cancelParentActions);
 
         if (empty($this->mountedActions)) {
             $this->defaultTableAction = null;
